webUI/react/src/components/reader/pager/VerticalPager.tsx

```diff
@@ -52,17 +52,33 @@ export default function VerticalReader(props: IReaderProps) {
         }
     }
 
+    function goLeft() {
+        if (settings.readerDirection === 'LeftToRight') {
+            prevPage();
+        } else {
+            nextPage();
+        }
+    }
+
+    function goRight() {
+        if (settings.readerDirection === 'LeftToRight') {
+            nextPage();
+        } else {
+            prevPage();
+        }
```

Comment:
`LeftToRight` and `RightToLeft` for VerticalPager? doesn't make sense?

you'll probably want this in Horizantal?